.travis.yml

```diff
@@ -3,8 +3,10 @@ sudo: false
 language: go
 
 go:
-- 1.7.5
-- 1.8
+- 1.8.x
```

Comment:
Now that 1.9 is out, do you want to just do `1.x` or also add `1.9.x`?

Reply:
Looks like we still need 1.9 in our tools.